src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -417,7 +417,15 @@ private async Task<(long, DatabaseTopology, List<string>)> CreateDatabase(string
             await ServerStore.WaitForCommitIndexChange(RachisConsensus.CommitIndexModification.GreaterOrEqual, newIndex);
 
             var members = (List<string>)result;
-            await WaitForExecutionOnRelevantNodes(context, name, clusterTopology, members, newIndex);
+            try
+            {
+                await ServerStore.WaitForExecutionOnRelevantNodesAsync(context, members, newIndex);
+            }
+            catch (Exception e) when (e is AggregateException)
```

Comment:
same

Reply:
this `try/catch` always repeat itself, maybe worth moving this into `WaitForExecutionOnRelevantNodesAsync`?

Reply:
I've considered this, but the message text for `throw new InvalidDataException()` changes depending on where we were waiting for execution on relevant nodes. Of course, we could move this inside `WaitForExecutionOnRelevantNodesAsync()`, but then we would lose the informativeness of the message.